server/includes/parser.hpp

```diff
@@ -0,0 +1,52 @@
+/**
+ * @file parser.hpp
+ * @brief 
+ * @date 26-12-2023
+ */
+
+// message example:x:0-y:10-hp:40-ip:127.0.0.1:13151
```

Comment:
It would be cool to have a better documented norm for the messages (maybe in the README? or a separate document)

Reply:
I think it would be useful for the review tomorrow

Reply:
I agree, on it